src/webgpu/api/operation/sampling/anisotropy.spec.ts

```diff
@@ -0,0 +1,15 @@
+export const description = `
+Tests the behavior of anisotropic filtering.
+
+Note that anisotropic filtering is never guaranteed to occur, but we might be able to test that
+no *more* than the provided maxAnisotropy samples are used, by testing how many unique
+sample values come out of the sample operation.
```

Comment:
Or have a test that check the result with and without anisotropy are different, or that aniso is done in the correct direction (by having a 2D gradient and checking we get more of the color in the correct direction). There's a bunch of implementation dependent behavior we'll still want to test, so maybe it is worth having some sort of soft expectations or non-failing expectations for them?